docs/develop/basics.md

```diff
@@ -0,0 +1,117 @@
+
+<SdkExampleTabs snippetId={'API_ConfigurationInRegionLowLatency'} />
+
+### Lambda
+
+`Configurations.Lambda` is a configuration that is available in some SDKs, and is optimized for the AWS Lambda environment. It has some configuration settings designed to pre-warm the client on Lambda cold starts, and to ensure the connection is re-established proactively if a Lambda remains idle for long enough for the connection to time out.
+
+<SdkExampleTabs snippetId={'API_ConfigurationILambda'} />
+
+If you do need to customize your configuration beyond what our pre-builts provide, you can build your own `Configuration`
+object.  See the examples in the `Configurations` namespace in the source code of your SDK  to see how they are constructed.
+
+We hope that these configurations will meet the needs of most users, but if you find them lacking in any way, please
+open a github issue, or contact us at `support@momentohq.com`. We would love to hear about your use case so that we
+can fix or extend the pre-built configs to support it.
+
+## Constructing a Cache Client
+
+The `CacheClient` is the main object that you will use to interact with a Momento Cache.  To instantiate one, you need to pass a `CredentialProvider`, a `Configuration`, and a default TTL.  The default TTL determines how long items will be stored in the cache if you don't specify a TTL explicitly when you save them.
+
+Here is an example of how to construct a `CacheClient`:
+
+<SdkExampleTabs snippetId={'API_InstantiateCacheClient'} />
+
+## Error Handling / Production Readiness
+
+Each of our SDKs has its own page in this documentation; you can navigate to them by clicking `Develop`->`SDKs` in the left nav.  On each of these pages you will find a link for "Taking your code to prod" that provides best practices for production-ready code in that language.
+
+Here are some general notes on error handling in Momento that apply to all SDKs.
+
+Errors that occur in calls to `CacheClient` methods are surfaced to developers as part of the return values of the calls, as opposed to by throwing exceptions. This makes them more visible at the time you are writing your code, and allows your IDE to be more helpful in ensuring that you've handled the ones you care about. (For more on our philosophy about this, see our blog post on why [Exceptions are bugs](https://www.gomomento.com/blog/exceptions-are-bugs). And send us any feedback you have!)
+
+This also helps to ensure that your application doesn't crash at runtime. Momento is a cache, so applications will usually have a fallback data source that they can retrieve data from if the cache is unavailable. Therefore, Momento SDKs are designed to avoid throwing exceptions so that your app won't crash if you forget to add a try/catch block. 
```

Comment:
```suggestion
This also helps to ensure that your application doesn't crash at runtime. Momento is a cache, so applications usually have a fallback data source they retrieve data from if the cache is unavailable. Therefore, Momento SDKs are designed to avoid throwing exceptions so your app won't crash if you forget to add a try/catch block. 
```